charts/tekton-pipelines/templates/common/tasks/slack-notification.yaml

```diff
@@ -102,13 +102,26 @@ spec:
         argo_app_url=$(echo $argo_app_url | sed 's/\/\//\//g')
         argo_app_url="https://$argo_app_url"
 
+        # Preparing jira links
+        # We are using grep as ash doesn't support regex test
+        jira_task=$(echo "$(params.head_commit_message)" | grep -oE '[A-Z0-9]+-[0-9]+' | head -n 1 )
```

Comment:
What if jira task ID would be in lower case? 
![image](https://github.com/saritasa-nest/saritasa-devops-helm-charts/assets/26448672/0e78d7c3-3e34-4c56-b21a-aa34758b0748)

Reply:
Than that string will be ignored.
Task was about exact task mention. If we use lower case that there will be a lot of false matches

Reply:
Let's ask other teams, i.e. Unity, iOS, Android to use python's team pre-commit hook then?